tests/roots/test-markup-rubric/index.rst

```diff
@@ -5,3 +5,35 @@ test-markup-rubric
 
 .. rubric:: This is
    a multiline rubric
+
+.. rubric:: A rubric with a class
+   :class: myclass
+
+.. rubric:: A rubric with a heading level 1
+   :level: 1
+   :class: myclass
+
+.. rubric:: A rubric with a heading level 2
+   :level: 2
+   :class: myclass
+
+.. rubric:: A rubric with a heading level 3
+   :level: 3
+   :class: myclass
+
+.. rubric:: A rubric with a heading level 4
+   :level: 4
+   :class: myclass
+
+.. rubric:: A rubric with a heading level 5
+   :level: 5
+   :class: myclass
+
+.. rubric:: A rubric with a heading level 6
+   :level: 6
+   :class: myclass
+
+.. rubric:: A rubric with a heading level 7
+   :level: 7
+   :class: myclass
```

Comment:
This only tests that the directive is rejecting numbers outwith 1-6, we need to test the writers independently.